src/main/java/com/kryptnostic/conductor/rpc/ResultSetAdapterFactory.java

```diff
@@ -75,12 +90,21 @@ public static Entity mapRowToEntity( Row row, Set<PropertyType> properties ) {
 	public static SetMultimap<FullQualifiedName, Object> mapRowToObject( Row row, Collection<PropertyType> properties ) {
 		SetMultimap<FullQualifiedName, Object> map = HashMultimap.create();
 		properties.forEach(property -> {
-			Object value = row.getObject( "value_" + property.getTypename() );
+			Object value = getObject( row, CassandraEdmMapping.getCassandraType( property.getDatatype() ), "value_" + property.getTypename() );
```

Comment:
We probably can revamp this a little bit - the main reason is that when @Bloodyhacker creates the cache table, he prefixes the column names by `value_`
https://github.com/dataloom/kindling/blob/develop/src/main/java/com/kryptnostic/sparks/ConductorSparkImpl.java#L372